File: FuramaResort/src/Models/Customer.java
package Models;

public class Customer {
    private String nameCustomer;
    private String dateOfBirth;
    private String gender;
    private int idCustomer;
    private int telephoneNumber;
    private String email;
    private String typeCustomer;
    private String address;
    private Services usingService;

    public Customer() {
    }

    public Customer(String nameCustomer, String dateOfBirth, String gender, int idCustomer, int telephoneNumber, String email, String typeCustomer, String address) {
        this.nameCustomer = nameCustomer;
        this.dateOfBirth = dateOfBirth;
        this.gender = gender;
        this.idCustomer = idCustomer;
        this.telephoneNumber = telephoneNumber;
        this.email = email;
        this.typeCustomer = typeCustomer;
        this.address = address;
    }

    public String getNameCustomer() {
        return nameCustomer;
    }

    public void setNameCustomer(String nameCustomer) {
        this.nameCustomer = nameCustomer;
    }

    public String getDateOfBirth() {
        return dateOfBirth;
    }

    public void setDateOfBirth(String dateOfBirth) {
        this.dateOfBirth = dateOfBirth;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public int getIdCustomer() {
        return idCustomer;
    }

    public void setIdCustomer(int idCustomer
    ) {
        this.idCustomer = idCustomer;
    }

    public int getTelephoneNumber() {
        return telephoneNumber;
    }

    public void setTelephoneNumber(int telephoneNumber) {
        this.telephoneNumber = telephoneNumber;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getTypeCustomer() {
        return typeCustomer;
    }

    public void setTypeCustomer(String typeCustomer) {
        this.typeCustomer = typeCustomer;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public Services getUsingService() {
        return usingService;
    }

    public void setUsingService(Services usingService) {
        this.usingService = usingService;
    }

    public String showInformation() {
        return "Customer {" + "Name: " + getNameCustomer() + "\n"
                    + "Date of Birth: " + getDateOfBirth() + "\n"
                    + "Gender: " + getGender() + "\n"
                    + "Id Number: " + getIdCustomer() + "\n"
                    + "Telephone Number: " + getTelephoneNumber() + "\n"
                    + "Email: " + getEmail() + "\n"
                    + "Type of Customer: " + getTypeCustomer() + "\n"
                    + "Address: " + getAddress() + "\n"
                    + "Service: " + getUsingService();
    }

    @Override
    public String toString() {
        return  nameCustomer + ',' +
                dateOfBirth + ',' +
                gender + ',' +
                idCustomer + ',' +
                telephoneNumber + ',' +
                email + ',' +
                typeCustomer + ',' +
                address + ',' +
                usingService;
    }
}

File: module5/test/Test-module5/src/app/module/customer.ts
export class Customer {
  private _id: number;
  private _nameCustomer: string;
  private _gender: number;
  private _birthday: string;
  private _sdt: string;
  private _typeCustomer: string;


  constructor(id: number, nameCustomer: string, gender: number, birthday: string, sdt: string, typeCustomer: string) {
    this._id = id;
    this._nameCustomer = nameCustomer;
    this._gender = gender;
    this._birthday = birthday;
    this._sdt = sdt;
    this._typeCustomer = typeCustomer;
  }


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get nameCustomer(): string {
    return this._nameCustomer;
  }

  set nameCustomer(value: string) {
    this._nameCustomer = value;
  }

  get gender(): number {
    return this._gender;
  }

  set gender(value: number) {
    this._gender = value;
  }

  get birthday(): string {
    return this._birthday;
  }

  set birthday(value: string) {
    this._birthday = value;
  }

  get sdt(): string {
    return this._sdt;
  }

  set sdt(value: string) {
    this._sdt = value;
  }

  get typeCustomer(): string {
    return this._typeCustomer;
  }

  set typeCustomer(value: string) {
    this._typeCustomer = value;
  }
}
